grammar.js

```diff
@@ -64,17 +75,17 @@ module.exports = grammar({
       $._phpunit_tag
     ),
 
-    // @inheritDoc is inline only, has no description
-    inline_tag: $ => prec(-1, seq(
```

Comment:
I don't think `prec()` is required here. Maybe you could explain why it was put there?

Reply:
This has also been in place since before I got here. If it's not needed, be gone w/ it!